tests/fitting/test_fitting_flows.py

```diff
@@ -156,16 +157,16 @@ def test_mlip_fit_maker_with_kwargs(
     # Test to check if gap fit runs with default hyperparameter sets (i.e. include_two_body and include_soap is True)
     gapfit = MLIPFitMaker().make(
         species_list=["Li", "Cl"],
-        isolated_atoms_energy=[-0.28649227, -0.25638457],
```

Comment:
Why is this removed and not renamed?

Reply:
First, there's no need to provide the energies of isolated atoms in the gap fitting since it's already included in the training data. Secondly, the format should be a dictionary instead of a list.

Reply:
Good